documentation/manual/src/main/asciidoc/modules/infinispan.asciidoc

```diff
@@ -661,11 +661,125 @@ public class News {
 |===
 ====
 
-====== Identifier generation strategies
+[[infinispan-embedded-id-generation-strategies]]
+
+====== Identifier generation strategies for Infinispan Embedded
+
+The Infinispan Embedded dialect will use
+http://infinispan.org/docs/stable/user_guide/user_guide.html#clustered_counters[clustered counters]
+for the generation of ids and sequences.
+
+Hibernate OGM will create counters when needed and use a default configuration if one is not
```

Comment:
"counters": in case the user doesn't know about Infinispan Counters this sentence might be confusing. Clarify you're referring to the specific Infinispan feature?